choccy_chip/src/emulator/emulator.rs

```diff
@@ -158,6 +158,21 @@ impl Emu {
     pub(crate) fn set_sound_timer(&mut self, val: u8) {
         self.special_registers.sound_timer = val;
     }
+
+    /// Loads a rom into ram.
+    ///
+    /// # Arguments
+    /// * `start`: The address in ram to store a copy of the rom at.
+    /// * `rom_bytes`: The rom to load into ram.
+    pub fn load_rom(&mut self, start: u16, rom_bytes: &[u8]) {
```

Comment:
Here we would be loading in a Rom as a type, which will let anyone who reads our library know how the API is supposed to used. Otherwise, they can just input any u8.

Reply:
I agree.